packages/components/src/mobile/image/style.native.scss

```diff
@@ -171,3 +171,9 @@
 .wide {
 	width: 100%;
 }
+
+.nonVisibleImage {
+	height: 0;
+	width: 0;
```

Comment:
> height and width must be non-zero or else onLoad does not fire on Android 

My understand is that using `0` for these values is acceptable because the above inline comment sourced from the [workaround](https://github.com/DylanVann/react-native-fast-image/issues/747#issuecomment-1264047430) does not apply as we do _not_ use FastImage for Android. Correct?

Reply:
For posterity and anyone finding this thread in the future, it turns out that a value of at least `1` must be used for both iOS and Android in order for the `onLoad` event to fire from FastImage.